src/bindings/build.py

```diff
@@ -34,6 +34,17 @@
     with open(header, "r") as hfile:
         ffi.cdef(hfile.read())
 
+# we need to easily control a few values for Windows builds
+source = []
+if os.getenv("PYNACL_SODIUM_STATIC") is not None:
+    source.append("#define SODIUM_STATIC")
```

Comment:
Can you add a comment explaining that what this does is cause `libsodium` to be a static lib... or whatever this define does?